images/oc-build-deploy-dind/openshift-templates/mongo/deployment.yml

```diff
@@ -71,7 +71,7 @@ objects:
           branch: ${SAFE_BRANCH}
           project: ${SAFE_PROJECT}
         annotations:
-          appuio.ch/backupCommand: /bin/sh -c "/bin/busybox tar -cf - -C /data/db ."
+          appuio.ch/backupCommand: /bin/sh -c "tar -cf - -C /data/db ."
```

Comment:
@cdchris12 
are these changes expected? seems to me like it's another issue?

Reply:
We had a discussion a while ago about the use of `tar` vs `/bin/busybox tar` and came to the conclusion that either one would work for us. It's more compatible to let sh use whatever `tar` binary is installed on the system, as busybox isn't installed in every container